python/cudf/cudf/core/column/string.py

```diff
@@ -2086,6 +2086,12 @@ def to_array(self, fillna=None):
 
         return self.to_arrow().to_pandas().values
 
+    def __array__(self, dtype=None):
+        raise NotImplementedError(
+            "__array__ is not supported in CuDF, \
+                To construct a host array, consider using .to_array()"
```

Comment:
I think something like this would be good:
```suggestion
            "Implicit conversion to a host NumPy array via __array__ is not allowed, \
                To explicitly construct a host array, consider using .to_array()"
```

Reply:
@kkraus14, Thanks. Also updated rest of the error messages similarly.